core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -50,8 +49,12 @@ public String encode(Barcodable barcodable) {
         sb.append("T 29,2,0,5,pt4;[DATE]\n");
         appendTruncated(sb, 20, alias);
 
-        sb.append("T 17,8,0,5,pt6;").append(LimsUtils.unicodeify(alias)).append("\n");
-        sb.append("T 17,11,0,5,pt6;").append(LimsUtils.unicodeify(name)).append("\n");
+        sb.append("T 17,8,0,5,pt6;");
+        appendBradyEscapedUnicode(sb, alias);
+        sb.append("\n");
+        sb.append("T 17,11,0,5,pt6;");
+        appendBradyEscapedUnicode(sb, name);
+        sb.append("\n");
         sb.append("A 1\n");
       } catch (UnsupportedEncodingException e) {
         log.error("get raw state", e);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "get raw state" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)